### PR TITLE
Tailored Flows: Link in bio processing step

### LINKED ISSUE #66561: Signup: Add processing step for link in bio
### Details

Add processing step on link in bio flow.

### Checklist

_No response_

### Related

_No response_

### PR DESCRIPTION
**Pull request by renancarvalho**
## Proposed Changes

Add processing step for link in bio flow.

image

image

## Testing instructions

- Pull this branch and run `yarn start` from the root folder
- Navigate to `http://calypso.localhost:3000/setup/intro?flow=link-in-bio&siteSlug=YOURSITE.wordpress.com`
- Advance on the steps by filling the fields
- After clicking on continue, you should see the processing step with the designs above.

Related to #66561 
Fixes #66561

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/index.ts [hunk 1]
```diff
@@ -80,5 +81,6 @@ export type StepPath =
 	| 'completingPurchase'
 	| 'intro'
 	| 'launchpad'
+	| 'linkInBioProcessingStep'
```
Comment (escapemanuele) [inline client/landing/stepper/declarative-flow/internals/steps-repository/index.ts]: I think that, to maintain convention, we should have `linkInBioProcessing`!

File: client/landing/stepper/declarative-flow/link-in-bio.ts [hunk 2]
```diff
@@ -15,12 +22,92 @@ export const linkInBio: Flow = {
...
+			}
+
+			// convert ArrayBuffer to Blob
+			const blob = new Blob( [ buffer ], { type: type } );
+
+			return blob;
+		};
+
+		const flowEnd = () => {
+			if ( site ) {
```
Comment (escapemanuele) [inline client/landing/stepper/declarative-flow/link-in-bio.ts]: We don't have site, when we submit from `linkInBioSetup`. So we should remove it from here, as well as inside the setup step :)

File: client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx [hunk 3]
```diff
@@ -76,7 +76,7 @@ const ProcessingStep: Step = function ( props ): ReactElement | null {
 		return () => clearTimeout( timeoutReference );
 	}, [ simulatedProgress, progress, __ ] );
 
-	const isJetpackPowered = props.flow === 'newsletters';
+	const isJetpackPowered = props.flow === 'newsletters' || props.flow === 'link-in-bio';
```
Comment (escapemanuele) [inline client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx]: We could do something like
```js
const isJetpackPowered = [ 'newsletters', 'link-in-bio' ].includes( props.flow );
```

**Review by escapemanuele — APPROVED** [review]
Thank you for this! I left only a minor comment that you can address or not, then you can deploy this :)

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7476941

Thank you @renancarvalho for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7476941).

**Pull request merged**